e2e2/test/cases/nvidia/mpi_test.go

```diff
@@ -15,20 +15,26 @@ import (
 	"sigs.k8s.io/e2e-framework/pkg/envconf"
 	"sigs.k8s.io/e2e-framework/pkg/features"
 
-	appsv1 "k8s.io/api/apps/v1"
 	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 var (
 	//go:embed manifests/mpi-job-pytorch-training-single-node.yaml
 	mpiJobPytorchTrainingSingleNodeManifest []byte
-	//go:embed manifests/mpi-job-pytorch-training-multi-node.yaml
-	mpiJobPytorchTrainingMultiNodeManifest []byte
-	//go:embed manifests/efa-device-plugin.yaml
-	efaDevicePluginManifest []byte
+	//go:embed manifests/mpi-job-nccl-test-multi-node.yaml
+	mpiJobNcclTestMultiNodeManifest         []byte
+	renderedMpiJobNcclTestMultiNodeManifest []byte
 )
 
+type ncclTestManifestTplVars struct {
+	WorkerNodeCount   int
+	GpuCount          int
+	EfaImage          string
```

Comment:
nit: calling this `EfaImage` feels a little odd, it could just be `TestImage` or `ContainerImage` because theres more than just EFA as a dependency

Reply:
Make sense to me, will change it to `NcclTestImage`